data/json/recipes/recipe_others.json

```diff
@@ -3669,6 +3669,32 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 50 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "scrap", 3 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "railroad_rail_small",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "1 h",
+    "autolearn": true,
+    "using": [ [ "steel_standard", 10 ] ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 }, { "id": "CHISEL", "level": 3 } ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 50 ], [ "oxy_torch", 10 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "railroad_track_small",
```

Comment:
Rather than craft rails, craft track, place track, I'd rather either do:
Craft rails, place rails (this would take rails, crosspieces, and a bunch of fasteners as inputs).
OR
Craft track, place track

The intermediate thing might be a better workflow IRL, but it just makes more keypresses here.

Reply:
merged the railroad rail recipe into the railroad track recipe